opal/mca/threads/argobots/configure.m4

```diff
@@ -56,8 +56,10 @@ AC_DEFUN([OPAL_CONFIG_ARGOBOTS_THREADS],[
 
     AS_IF([test $opal_argo_happy = yes && test $opal_argo11_happy = no],
           [AC_MSG_ERROR([Open MPI requires Argobots 1.1 or newer.])])
-    AS_IF([test $opal_argo_happy = yes && test $opal_argo11_happy = yes && test -n "$opal_argo_dir"],
-          [OPAL_ARGO_INCLUDE_PATH="$opal_argo_dir/include/"],
+dnl extract the fully qualified include path from the opal_argo_CPPFLAGS variable
+dnl for use in threads_argobots.h
+    AS_IF([test $opal_argo_happy = yes && test $opal_argo11_happy = yes],
+          [OPAL_ARGO_INCLUDE_PATH=`echo "${opal_argo_CPPFLAGS}/" | sed -e 's/\-I//g' | sed -e 's/ //g'`],
```

Comment:
This has an implicit assumption that opal_argo_CPPFLAGS only contains 1 directory, which may be true today but doesn't seem long term true.

But why are we doing this at all?  It looks like we're doing it to because we need to include `abt.h` in a header that then is included in `opal/mca/threads/threads.h`.  I think we'd be better off skipping this bit (that doesn't do what we test) and just adding opal_argo_CPPFLAGS into CPPFLAGS.